### PR TITLE
Basic Email template applied + Wrapping email service added

### PR DESCRIPTION
**Pull request by machador**
https://www.pivotaltracker.com/story/show/108122002

### CODE REVIEW

File: grails-app/views/layouts/organizationEmailLayout.gsp [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <title>CHIME</title>
```
Comment (ascuccimarra) [inline grails-app/views/layouts/organizationEmailLayout.gsp]: Nooo eso queda re feo

File: grails-app/views/registration/emails/_newUserInvitation.gsp [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Hi,
 <br><br>
 You have been invited to join Pregnancy Control App by ${inviterName}.
```
Comment (ascuccimarra) [inline grails-app/views/registration/emails/_newUserInvitation.gsp]: Shoul be SimpleCare I think

File: grails-app/views/layouts/organizationEmailLayout.gsp (hunk 1 above)
Comment (machador) [inline grails-app/views/layouts/organizationEmailLayout.gsp]: No idea where that came from (?)

File: grails-app/views/registration/emails/_newUserInvitation.gsp (hunk 2 above)
Comment (machador) [inline grails-app/views/registration/emails/_newUserInvitation.gsp]: That was already there, but I will change it as it is an old email that has not been updated.

**Pull request merged**